WordPress/Classes/ViewRelated/Post/AztecPostViewController.swift

```diff
@@ -201,13 +212,15 @@ class AztecPostViewController: UIViewController {
         return context
     }()
 
+
     // MARK: - Lifecycle Methods
 
     init(post: AbstractPost) {
         self.post = post
 
         super.init(nibName: nil, bundle: nil)
 
+        self.shouldRemovePostOnDiscard = shouldRemoveOnDismiss(post: post)
```

Comment:
Remove on dismiss or discard?

My 2 cents: change the method name to`shouldRemoveOnDiscard`.